stories/FileTree.stories.tsx

```diff
@@ -60,10 +69,90 @@ const version = {
         filename: 'styles.css',
         path: 'styles.css',
       },
+      'jquery-ui': {
+        ...fakeVersionEntry,
```

Comment:
I'd like to see this (and other directory entries) switched to a common base to cut down on repetition. For example:

```diff
diff --git a/stories/FileTree.stories.tsx b/stories/FileTree.stories.tsx
index ef3aede..836cb17 100644
--- a/stories/FileTree.stories.tsx
+++ b/stories/FileTree.stories.tsx
@@ -21,6 +21,11 @@ import { renderWithStoreAndRouter } from './utils';
 
 const versionId = 456;
 
+const fakeDirectoryEntry = {
+  ...fakeVersionEntry,
+  mime_category: 'directory' as VersionEntryType,
+};
+
 const version = {
   ...fakeVersion,
   id: versionId,
@@ -33,10 +38,9 @@ const version = {
         path: 'manifest.json',
       },
       'background-scripts': {
-        ...fakeVersionEntry,
+        ...fakeDirectoryEntry,
         filename: 'background-scripts',
         path: 'background-scripts',
-        mime_category: 'directory' as VersionEntryType,
       },
       'background-scripts/index.js': {
         ...fakeVersionEntry,
```